Purchases/Classes/Public/RCPurchases.m

```diff
@@ -560,4 +571,41 @@ - (void)updateOriginalApplicationVersion
     }];
 }
 
+- (void)createAlias:(NSString *)alias
+{
+    [self createAlias:self.appUserID completion:^(NSError * _Nullable error) {}];
+}
+
+- (void)createAlias:(NSString *)alias completion:(void (^)(NSError * _Nullable error))completion
+{
+    [self.backend createAliasForAppUserID:self.appUserID withNewAppUserID:alias completion:^(NSError * _Nullable error) {
+        if (error == nil) {
+            [self identify:alias];
+        }
+        if (completion != nil) {
+            completion(error);
+        }
+    }];
+}
+
+- (void)identify:(NSString *)appUserID
+{
+    [self.userDefaults removeObjectForKey:RCAppUserDefaultsKey];
+    self.appUserID = appUserID;
+    self.isUsingAnonymousID = NO;
```

Comment:
We need to talk about `isUsingAnonymousID` here. Some people will want to use it with an identified user. 

We might want to change the name of it.